index.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable */
```

Comment:
- [ ] you shouldn't disable the eslint it ensures that you have a great, clean code so please try to remove the that line try to solve the issues